src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,117 @@
-const Countdown = () => null;
+import React, { useState, useEffect } from 'react';
 
-export default Countdown;
+// Countdown starts at 2 minutes and 30 seconds (150000 milliseconds) and counts down to 0
+const Countdown: React.FC = () => {
+  const INITIAL_TIME = 150000; // 2 minutes and 30 seconds in milliseconds
+  const [time, setTime] = useState(INITIAL_TIME);
+  const [isRunning, setIsRunning] = useState(false);
+
+  useEffect(() => {
+    let intervalId: number;
+    if (isRunning && time > 0) {
+      intervalId = setInterval(() => {
+        setTime(prevTime => {
+          if (prevTime <= 10) {
+            setIsRunning(false);
+            return 0;
+          }
+          return prevTime - 10;
+        });
+      }, 10);
```

Comment:
10ms is fast to be changing the interval... 100ms would be better for all of these intervals. Does somebody exercising care for sub-100ms precision? Probably not.